website/blog/2022-05-10-acf-in-a-project.md

```diff
@@ -0,0 +1,140 @@
+---
+title: ACF in a project
+description: Example of using ACF in your project
+slug: acf-in-a-project
+authors: obradovic
+date: 2022-05-10
+tags: [eightshift, boilerplate, acf, advanced custom fields, theme options]
+hide_table_of_contents: false
+---
+
+If you've worked with WordPress for long enough, you've heard of the **_Advanced Custom Fields_** plugin (**_ACF_** for short). While the use of blocks has simplified content editing, making meta fields less of a necessity, there are still cases in which meta fields are very useful.
+<!--truncate-->
+
+## Ways of registering ACF fields
+
+There are multiple approaches to ACF field registration. The easiest way to register fields is using the WP Admin interface. However, if you're using Git, this may not be the best way to do it since the fields are being saved in the database, which means you would have to export and import fields across environments each time you make a change.
+
+We recommend registering the fields using PHP, which will allow you to version your field definitions using git. There is also an option to register the fields with **_.json_** files, but in Eightshift Development kit, we already have some goodies which will make field registration a breeze.
+
+## Exporting PHP code
+
+The approach we use for registering ACF fields is by adding those fields through the WP Admin ACF interface first. As an example, I'll add a field group called **_Intro_**, which will contain one text field of the same name. That field group will be displayed only on posts. You can add a few more fields, but for the sake of simplicity, I'll only use one field.
+
+When you define all the fields you need, save them and go to `Custom Fields -> Tools`. Here you will see an option to **_Export Field Groups_**. Simply select the field groups which you want to export and select **_Generate PHP_**.
+
+![ACF PHP code export](/img/blog/acf-generate-php.png)
+
+This will generate a PHP code snippet that you can use in your theme. But now you may be wondering, where exactly should that code go?
```

Comment:
```suggestion
This will generate a PHP code snippet that you can use in your theme. Now you may be wondering, where exactly should that code go?
```